### PR TITLE
export converter as a usable package

### PR DESCRIPTION
**Pull request by imeoer**
```
So that we can import the `pkg/converter` package into other
projects to integrate the image conversion ability.

This patch is no logic changes.
```

### CODE REVIEW

File: pkg/converter/converter.go [hunk 1]
```diff
@@ -132,32 +101,19 @@ func (cvt *LocalConverter) Convert(ctx context.Context, source string) error {
 
 	logger.Infof("converting image %s", source)
 	start = time.Now()
-	desc, err := cvt.driver.Convert(ctx, content)
+	desc, err := cvt.driver.Convert(ctx, cvt.provider)
 	if err != nil {
 		return errors.Wrap(err, "convert image")
 	}
-
```
Comment (liubogithub) [inline pkg/converter/converter.go:139]: Why is this removed?

**Review by liubogithub — APPROVED** [review]

File: pkg/converter/converter.go (hunk 1 above)
Comment (imeoer) [reply pkg/converter/converter.go:139]: We need to migrate to the new artifact reference spec, the annotations defined here are unnecessary anymore.
